src/main/res/layout/reactions_pill.xml

```diff
@@ -12,11 +12,13 @@
     android:paddingEnd="7dp"
     android:gravity="center">
 
-  <org.thoughtcrime.securesms.components.emoji.EmojiTextView
+  <!-- to avoid default transparency in text color that makes emoji transparent -->
```

Comment:
took me a moment to get, what the comment is about - and that even though the changed line was hilited on github :)

i suggest to either remove the comment or say:

```suggestion
  <!-- android:textColor is set to avoid default transparency making emoji transparent -->
```

Reply:
oops! yes, the reason was that the comment was initially right above the android:textColor line but that caused a compilation error due to not allowed in xml to put comments inside a tag declaration and I moved it outside to avoid the error and keep testing and forgot to update the comment, thanks a lot for pointing out!